cli/cmd/plugin/ui/web/tanzu-ui/src/shared/components/UnmanagedClusterInfo/UnmanagedClusterInfo.scss

```diff
@@ -0,0 +1,3 @@
+label {
+    color: grey;
```

Comment:
I'd suggest re-using a Clarity grey, like in index.scss: .text-context-label {
    color: var(--cds-global-color-gray-700);

Reply:
Good point! Justin mentioned to use clarity options but it slipped my mind thank you!